src/mongo/initDb.js

```diff
@@ -1,16 +1,14 @@
 const mongoose = require('mongoose');
+const databaseManager = require('./databaseManager');
 
 const databaseInit = () => {
-    mongoose.connect('mongodb://localhost:27017/', {
-        useNewUrlParser: true,
-        dbName: "portafolio",
-        connectTimeoutMS: 5000
-    });
+    const mongoDatabase = new databaseManager(
```

Comment:
Any reason to use this databaseManager instead of mongoose.connect directly?

Reply:
Allows you to connect to multiple databases if needed, by just creating another instance of a DatabaseManager

Reply:
Cool! Maybe we can add later something to close connection